src/WebJobs.Script.Grpc/GrpcServiceCollectionsExtensions.cs

```diff
@@ -16,6 +16,9 @@ public static IServiceCollection AddScriptGrpc(this IServiceCollection services)
 
             services.AddSingleton<IRpcServer, AspNetCoreGrpcServer>();
 
+            services.AddHttpForwarder();
+            services.AddSingleton<IHttpProxyService, DefaultHttpProxyService>();
```

Comment:
Unfortunately, we can't get this behind the feature flag (can't access `FeatureFlags` yet at this point, host will crash) so these services will be added but not used.